cluster/plugin/local/cmd/main.go

```diff
@@ -139,6 +138,20 @@ func main() {
 	}
 	destroyCmd.PersistentFlags().BoolVarP(&opts.ForceLeave, "force-leave", "", false, "force leave the swarm")
 
+	// override default value if env vars are set
```

Comment:
We should be using command options instead of environment variables. It would be better to align the CLI to supply the correct args instead.